assets/src/components/propertiesPanel/miniMap.tsx

```diff
@@ -0,0 +1,68 @@
+import { DomEvent } from "leaflet"
+import React, { useContext, useEffect } from "react"
+import { Link } from "react-router-dom"
+import { StateDispatchContext } from "../../contexts/stateDispatchContext"
+import { useStations } from "../../hooks/useStations"
+import { Vehicle, VehicleId } from "../../realtime"
+import { Shape, Stop } from "../../schedule"
+import { setSelectedVehicle } from "../../state/searchPageState"
+import inTestGroup, { MAP_BETA_GROUP_NAME } from "../../userInTestGroup"
+import { mapModeForUser } from "../../util/mapMode"
+import Map from "../map"
+
+const SearchMapLink = ({ vehicleId }: { vehicleId: VehicleId }) => {
+  const [_state, dispatch] = useContext(StateDispatchContext)
```

Comment:
**suggestion:** I think you can also do `const [, dispatch] = useContext(StateDispatchContext)` if you don't need to get the `state`.